tests/kuttl/tests/dataplane-deploy-global-service-test/00-assert.yaml

```diff
@@ -1,3 +1,14 @@
+apiVersion: kuttl.dev/v1beta1
+kind: TestAssert
+timeout: 600
```

Comment:
Isn't default timeout enough?

Reply:
the timeout in kuttl-test.yaml applies to all tests afaik